lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -257,7 +257,7 @@ def define
 
       outcome :outcome_marriage_france_or_french_overseas_territory
 
-      outcome :outcome_os_affirmation
+      outcome :outcome_opposite_sex_marriage_affirmation
```

Comment:
Done. I've renamed this in 76380e6acf6b2051e6eb15637685f4fb0d3d80b5 (in PR #2409).